### ISSUE TITLE
Option to truncate note titles in VerticalTabs sidebar

### ISSUE THREAD

**Issue by RobinLandy** [oxdc/obsidian-vertical-tabs#46]
Would it be possible to have an option to truncate note titles in the sidebar?

Many of my note titles use two lines in the VerticalTabs sidebar, which means that I can only see a few of the open tabs without vertical scrolling.

1. I'd like an option for the displayed titles to be truncated with "..." so that they only use one line, regardless of the width of the sidebar
2. The point at which the titles truncate would need to move if/when the sidebar is resized

**Comment by Lotus49b** [comment]
Plus 1 for this

**Comment by oxdc** [comment]
Thank you! I'm working on the settings view, this option will be available when it comes out. Before you get the release, you may temporarily use the following CSS snippet,

```css
.obsidian-vertical-tabs-container .tree-item-inner-text {
    white-space: nowrap;
}
```

**Comment by oxdc** [comment]
Hi @Lotus49b and @RobinLandy ! This feature is now available in v0.12.0.

**Comment by RobinLandy** [comment]
Just upgraded! Looks great. Thanks for all the hard work you're putting into this plugin.
